test/integration/targets/filesystem/tasks/overwrite_another_fs.yml

```diff
@@ -0,0 +1,44 @@
+- name: 'Recreate "disk" file'
+  command: 'dd if=/dev/zero of={{ dev }} bs=1M count={{ fssize }}'
+
+- name: 'Create a vfat filesystem'
+  command: 'mkfs.vfat {{ dev }}'
+  when: ansible_system != 'FreeBSD'
```

Comment:
Wouldn't it be more future-proof to use ansible_system == 'Linux' ? How well would it work on others BSD and/or windows ?

Reply:
Due to `aliases` file it won't be executed on Windows but it could be problematic with others BSD. Updated.